### PR TITLE
feat(toolbar): sass module refactor

### LINKED ISSUE #9093: Toolbar SASS Refactor
This issue is for updating the Notification to Sass Modules in the `@carbon/styles` package. In this refactor, we will:

- [ ] Copy over styles for for the component to `packages/styles/scss/components/toolbar/_pagination-nav/scss`
  - Any additional files will live in the folder `packages/styles/scss/components/toolbar//`
- [] Create an `_index.scss` at `packages/styles/scss/components/toolbar//_index.scss`
- [ ] Create a test file at `packages/styles/scss/components/__tests__/toolbar//-test.js`

The `_index.scss` should:

- `@forward` the mixin and any variables from `toolbar//.scss`
- Include the styles from `toolbar//.scss` by calling the mixin

The `toolbar/-test.js` file should test the following:

- The Public API of the `toolbar/_index.scss` file
- Any configuration options that are present for the file

The `_toolbar/.scss` will then need to be refactored to use the new import paths and conventions. You can find common recipes here: https://github.com/carbon-design-system/carbon/issues/8899#issuecomment-860887959

If your component includes **component tokens**, follow the instructions below 👇
<details>
  <summary>Component tokens</summary>

- Copy over the `_tokens.scss` file for the component to `scss/components/<component>/_tokens.scss`
- Change any tokens to their new paths, common changes that need to happen will be if the token map uses IDL colors or token colors. These now come from `scss/_colors.scss` and `scss/_theme.scss` specifically.
- Bring in `scss/utilities/_component-tokens.scss` file
- Use this utility to generate four exports in `_tokens.scss`: `$white`, `$g10`, `$g90`, and `$g100`
- Each export should look like the following: `$white: component-tokens.get-tokens($-tokens, themes.$white);`
- Add the tokens to the component exports in `scss/components/<component>/_index.scss` by using `@forward 'tokens'`
- If using storybook, update `packages/carbon-react/.storybook/styles.scss` to include the new component tokens for each theme. This will look like:

```scss
@use '../scss/components/notification';

:root {
  @include styles.theme(themes.$white, notification.$white);
}

.g10 {
  // ...
}
```

You may also need to add the entrypoint in the `@carbon/react` package itself at `packages/carbon-react/scss/components/<component>/_index.scss`
</details>

### LINKED ISSUE #9094: Unstable_Treeview SASS Refactor
This issue is for updating the Notification to Sass Modules in the `@carbon/styles` package. In this refactor, we will:

- [ ] Copy over styles for for the component to `packages/styles/scss/components/floating-menu/_pagination-nav/scss`
  - Any additional files will live in the folder `packages/styles/scss/components/pagination-nav/`
- [ ] Create an `_index.scss` at `packages/styles/scss/components/pagination-nav/_index.scss`
- [ ] Create a test file at `packages/styles/scss/components/__tests__/pagination-nav/-test.js`

The `_index.scss` should:

- `@forward` the mixin and any variables from `_pagination-nav/.scss`
- Include the styles from `_pagination-nav/.scss` by calling the mixin

The `pagination-nav/-test.js` file should test the following:

- The Public API of the `pagination-nav/_index.scss` file
- Any configuration options that are present for the file

The `_pagination-nav/.scss` will then need to be refactored to use the new import paths and conventions. You can find common recipes here: https://github.com/carbon-design-system/carbon/issues/8899#issuecomment-860887959

If your component includes **component tokens**, follow the instructions below 👇
<details>
  <summary>Component tokens</summary>

- Copy over the `_tokens.scss` file for the component to `scss/components/<component>/_tokens.scss`
- Change any tokens to their new paths, common changes that need to happen will be if the token map uses IDL colors or token colors. These now come from `scss/_colors.scss` and `scss/_theme.scss` specifically.
- Bring in `scss/utilities/_component-tokens.scss` file
- Use this utility to generate four exports in `_tokens.scss`: `$white`, `$g10`, `$g90`, and `$g100`
- Each export should look like the following: `$white: component-tokens.get-tokens($-tokens, themes.$white);`
- Add the tokens to the component exports in `scss/components/<component>/_index.scss` by using `@forward 'tokens'`
- If using storybook, update `packages/carbon-react/.storybook/styles.scss` to include the new component tokens for each theme. This will look like:

```scss
@use '../scss/components/notification';

:root {
  @include styles.theme(themes.$white, notification.$white);
}

.g10 {
  // ...
}
```

You may also need to add the entrypoint in the `@carbon/react` package itself at `packages/carbon-react/scss/components/<component>/_index.scss`
</details>

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #9093

### CODE REVIEW

**Comment by joshblack** [comment]
@andreancardona I _think_ toolbar is getting removed in v11 so we should be good dropping this

**Comment by andreancardona** [comment]
> @andreancardona I _think_ toolbar is getting removed in v11 so we should be good dropping this

yeah I saw it was deprecated...but wasn't sure 😭 so I ended up doing it haha

**Comment by tw15egan** [comment]
☠️

**Comment by andreancardona** [comment]
@joshblack should I just close 👀 lol

File: packages/carbon-react/src/components/Toolbar/Toolbar.stories.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  RadioButton,
+} from 'carbon-components-react';
+
+//  const toolbarProps = {
+//    className: 'some-class',
+//  };
+
+//  const inputProps = {
+//    className: 'some-class',
+//  };
```
Comment (emyarod) [inline packages/carbon-react/src/components/Toolbar/Toolbar.stories.js]: would these still be needed?

```suggestion
```

**Comment by joshblack** [comment]
@andreancardona go for it!

**Comment by andreancardona** [comment]
closing...since it isn't needed hehe :(

**Pull request closed without merging**